practiceapp/src/main/java/com/practiceapp/practiceapp/controller/profile/ProfileController.java

```diff
@@ -0,0 +1,49 @@
+package com.practiceapp.practiceapp.controller.profile;
+
+
+import com.practiceapp.practiceapp.entity.Profile;
+import com.practiceapp.practiceapp.service.ProfileService;
+import com.practiceapp.practiceapp.utils.PicturesApi;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(path = "/post")
```

Comment:
I couldn't understand why the path to the profiles is "/post" and it is already used by CreatePostController. I think "/profile" would be better.

Reply:
You are right, I forgot to change. 
Thanks